web/src/Components/Public/Submission/SubmissionHeader/SubmissionHeader.styles.jsx

```diff
@@ -0,0 +1,59 @@
+import {makeStyles} from '@material-ui/core/styles';
+
+export const useStyles = makeStyles((theme) => ({
+  submissionSummaryContainer: {
+    width: '100%',
+    height: '50px',
+    // backgroundColor: '#21262D',
+    backgroundColor: theme.palette.grey[900],
```

Comment:
I do not believe this exists :)

Reply:
Changed to theme.palette.dark.blue['200']